scss/themes/_branded.scss

```diff
@@ -53,11 +53,11 @@
 	position: absolute;
 	bottom: 0;
 	right: 0;
-	width: 100px;
+	width: 92px;
```

Comment:
These look an awful lot like ✨ magic numbers

Reply:
Ah, should they live in a magic box somewhere?

Reply:
Ah sorry gotcha...I think there are couple of magic numbers already in play.

1. `padding-right` on the standfirst
![screenshot 2018-12-05 at 16 45 54](https://user-images.githubusercontent.com/616321/49529140-4fffe080-f8ad-11e8-8b59-d155b12bcbe1.png)

2. `margin-left` on the image - think this stops the standfirst clobbering the image
![screenshot 2018-12-05 at 16 46 03](https://user-images.githubusercontent.com/616321/49529169-5ee69300-f8ad-11e8-8d89-23cd264e7710.png)

Dunno if there's a way to avoid avoid the clobbering without hardcoding something 🤔 ?

Reply:
@gvonkoss ok since that padding is applied to headline too, created a mixin. Reckon that's overkill or ok?